iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -783,6 +785,43 @@ public void testDropHiveTableWithoutUnderlyingTable() throws IOException {
     shell.executeStatement("DROP TABLE " + identifier);
   }
 
+  @Test
+  public void testAlterTableAddColumns() throws Exception {
+    Assume.assumeTrue("Iceberg - alter table/add column is only relevant for HiveCatalog",
+        testTableType == TestTables.TestTableType.HIVE_CATALOG);
+
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+
+    // Create HMS table with with a property to be translated
+    shell.executeStatement(String.format("CREATE EXTERNAL TABLE default.customers " +
```

Comment:
`TestTables` has several convenience methods to create tables with schema and spec. It might be useful here too